src/test/scala/com/microsoft/azure/synapse/ml/core/test/fuzzing/FuzzingTest.scala

```diff
@@ -175,6 +175,51 @@ class FuzzingTest extends TestBase {
     assertOrLog(classesWithoutFuzzers.isEmpty, classesWithoutFuzzers.mkString("\n"))
   }
 
+  test("Verify all stages can be tested in R") {
+    val exemptions: Set[String] = Set(
+      "com.microsoft.azure.synapse.ml.cognitive.DocumentTranslator",
+      "com.microsoft.azure.synapse.ml.automl.TuneHyperparameters",
+      "com.microsoft.azure.synapse.ml.train.TrainedRegressorModel",
+      "com.microsoft.azure.synapse.ml.vw.VowpalWabbitContextualBanditModel",
+      "com.microsoft.azure.synapse.ml.train.TrainedClassifierModel",
+      "com.microsoft.azure.synapse.ml.vw.VowpalWabbitClassificationModel",
+      "com.microsoft.azure.synapse.ml.isolationforest.IsolationForestModel",
+      "com.microsoft.azure.synapse.ml.nn.ConditionalKNNModel",
+      "com.microsoft.azure.synapse.ml.lightgbm.LightGBMClassificationModel",
+      "com.microsoft.azure.synapse.ml.core.serialize.TestEstimatorBase",
+      "com.microsoft.azure.synapse.ml.core.serialize.MixedParamTest",
+      "com.microsoft.azure.synapse.ml.featurize.CleanMissingDataModel",
+      "com.microsoft.azure.synapse.ml.stages.TimerModel",
+      "com.microsoft.azure.synapse.ml.featurize.DataConversion",
+      "com.microsoft.azure.synapse.ml.automl.TuneHyperparametersModel",
+      "com.microsoft.azure.synapse.ml.automl.BestModel",
+      "com.microsoft.azure.synapse.ml.nn.KNNModel",
+      "com.microsoft.azure.synapse.ml.vw.VowpalWabbitRegressionModel",
+      "com.microsoft.azure.synapse.ml.stages.ClassBalancerModel",
+      "com.microsoft.azure.synapse.ml.core.serialize.StandardParamTest",
+      "com.microsoft.azure.synapse.ml.core.serialize.ComplexParamTest",
+      "com.microsoft.azure.synapse.ml.featurize.ValueIndexerModel",
+      "com.microsoft.azure.synapse.ml.lightgbm.LightGBMRankerModel",
+      "com.microsoft.azure.synapse.ml.lightgbm.LightGBMRegressionModel",
+      "com.microsoft.azure.synapse.ml.cognitive.FormOntologyTransformer",
+      "com.microsoft.azure.synapse.ml.cognitive.DetectMultivariateAnomaly",
+      "com.microsoft.azure.synapse.ml.train.ComputePerInstanceStatistics"
```

Comment:
How does this compare to python exclusions?

Reply:
It's the same -- I just duped the code and tweaked it for R. But since you asked, I tested things and it turns out that DataConversion doesn't need to be included in either the python or R version.